flormcalvo/poc-csyd-mentor
# Dev 016

## Rationale

Click at any card should fetch the single card

## Advice for Reviewers & Testing Notes

1. npm install in the client directory as new npm package 'react-router-dom' has been added
2. run client / server locally
3. check out the functionality of the links, and views esp mobile view.

## Screenshots:

![image](https://user-images.githubusercontent.com/47847951/79748567-048a8d00-8351-11ea-8e64-0b7a8fb8ebcd.png)

## Task Name

- ask name from Notion: DEV-016 Fetch and display single mentor information

## Linting Checklist
The following is a list of linting rules that ESLint is not currently linting for. Please make sure your code conforms to this list. Examples/definitions of the rules can be found [here](https://hireup.atlassian.net/wiki/pages/viewpageattachments.action?pageId=618037301&metadataLink=true&preview=/618037301/618168365/PR_Checklist.pdf)
